README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# hw01
+---
+title: "Kacey Marioghae HW01"
```

Comment:
You actually don't need this header for this assignment. But you will in subsequent HWs!